Module_Java14/src/main/java/Java14/NewSwitchException.java

```diff
@@ -0,0 +1,30 @@
+package Java14;
+
+public class NewSwitchException {
+    public static void demo() {
+        System.out.println(java14(Month.NOV));
+    }
+
+    public static String java14(Month month) {
+        var result = switch (month) {
+            case JAN -> "january";
+            case FEB -> "february";
+            case MAR -> "march";
+            case APR -> "april";
+            case MAY -> "may";
+            case JUN -> "juny";
+            case JUL -> "july";
+            case AUG -> "august";
+            case SEP -> "september";
+            case OCT -> "october";
+            case NOV -> "november";
+            case DEC -> "december";
+
+            default -> null;
+        };
+        return result;
+    }
+    enum Month {
```

Comment:
Create separate class(enum) instead of inner

Reply:
> Create separate class(enum) instead of inner

Done